rust/template/distributed_datalog/src/accumulate/accumulator.rs

```diff
@@ -47,6 +47,10 @@ where
 
     /// Return the current state of the data.
     fn get_current_state(&self) -> HashMap<RelId, HashSet<V>>;
+
+    /// Sends a deletion update to all observers, thus clearing the accumulated state.
```

Comment:
I agree. Maybe we should also think of somehow "registering" the wish to clear the accumulator so that `clear()` is called on the next `commit()` automatically? This way, one can take a source offline while it is still committing updates. Otherwise, you would need to try multiple times to get the correct timing in-between transactions. What do you think @ryzhyk?

Reply:
I prefer to keep the semantics simple and have the control plane make sure that it does not call the method at the wrong time.  Delayed invocation sounds like a tricky thing to implement and use.

Reply:
Sure, that's also a valid point. I just think it will also be hard for the control plane to know exactly when an transaction is going on, as observables can call functions on their observers without sending any information about it to the control plane. Nevertheless, I completely agree with you to keep it simple in the first place.